tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -436,6 +441,40 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	k_sleep(K_MSEC(1000));
```

Comment:
I needed to increase the sim_length in the basic_advx.sh script to 5e7 for these tests to pass on my system. Please increase this value